test_integration/test_no_ssl_verification.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=missing-docstring
+import os
+import threading
+from http.server import HTTPServer, SimpleHTTPRequestHandler
+from ssl import PROTOCOL_TLS_SERVER, SSLContext
+
+from test.test_base_service import AnyServiceV1
+from ibm_cloud_sdk_core.authenticators import NoAuthAuthenticator
+
+
+def test_no_ssl_verification():
```

Comment:
Should we add a test that uses the same certificate and key, but does NOT disable ssl verification?   I assume this would be a negative test as we should get an exception for that scenario, right?

Reply:
Right and it's a good idea. I'll also add the command to generate the cert files.